### PR TITLE
Added technology and Repository tools. Issue: #3.

### PR DESCRIPTION
**Pull request by felix-zailskas**
Licensing information can be found here:
https://reuse.software/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,106 @@
...
+
+  - **epic**: It contains a module branch. Like high level of feature. For example, we have an authentication module then we can create a branch like "epic/authentication"
+
+    - **feature**: It contains specific features under the module. For example, under authentication, we have a feature called registration. Sample branch name: "feature/registration"
+
+    - **bugfix**: It contains bug fixing during the testing phase and branch name start with the issue number for example "bugfix/3-validate-for-wrong-user-name"
+
+### Commits and Pull Requests
+
+The stable branches `main` and `dev` are protected against direct pushes. To commit code to these branches create a pull request (PR) describing the feature/bugfix that you are committing to the `dev` branch. This PR will then be reviewed by another SD from the project. Only after being approved by another SD a PR may be merged into the `dev` branch. Periodically the stable code on the `dev` branch will be merged into the `main` branch by creating a PR from `dev`. Hence, every feature that should be committed to the `main` branch mus first run without issues on the `dev` branch for some time.
```
Comment (Tims777) [inline README.md:59]: Small typo here, that mus be fixed ;)

File: README.md [hunk 2]
```diff
@@ -1,12 +1,106 @@
...
+When doing pair programming be sure to always have all SDs mentioned in the commit message. Each SD should be listed on a new line for clarity reasons.
+
+```[bash]
+git commit -a -m "Fixed problem #123
+> Co-authored-by: Manu Musterperson <manu.musterperson@fau.de>"
+```
+
+### License
+
+This project is operated under an MIT license. Every file must contain the REUSE-compliant license and copyright declaration:
```
Comment (Tims777) [inline README.md:83]: What about binary files?

Comment (Tims777) [reply README.md:83]: Also, can you add a link here with more information regarding REUSE and SPDX?

**Review by Tims777 — CHANGES_REQUESTED** [review]
Good work so far!
Personally, I think that the license information at the beginning of each file is somewhat cryptic so please add a link with more information. I am sure that at least some of the others will also be curious about how this works :-)

File: README.md (hunk 1 above)
Comment (felix-zailskas) [reply README.md:59]: What exactly is the typo?

File: README.md (hunk 2 above)
Comment (felix-zailskas) [reply README.md:83]: Check the REUSE documentation for this. https://reuse.software/tutorial/#binary-and-uncommentable-files

File: README.md (hunk 1 above)
Comment (Tims777) [reply README.md:59]: "mus(t)"

File: README.md (hunk 2 above)
Comment (Tims777) [reply README.md:83]: "You may choose to drop items except the copyright holder, which must always be included. We recommend that you include all items, however." So I am afraid that we need to add a copyright holder to all files as well.

Comment (felix-zailskas) [reply README.md:83]: I am aware of that but unsure who will be the copyright holder. I will ask that in the lecture and add it afterwards.

Comment (Tims777) [reply README.md:83]: I think some link (maybe https://reuse.software/faq/) should be in the README for future reference. Can you do that? I don't want to run into merge conflicts with you :-)

File: README.md (hunk 1 above)
Comment (felix-zailskas) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Tims777) [reply README.md:83]: I think the copyright holder will be all of us / our team. So maybe we can just put "Sum Insights" there? But it definitely won't do any harm to ask in the lecture, thanks for that.

Comment (felix-zailskas) [reply README.md:90]: The copyright holder are most likely the authors of the files. Berkay and I have added them as the holders for now and I will ask again in the lecture.

**Review by rbbozkurt — APPROVED** [review]
Overall good work!

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+SPDX-FileCopyrightText: 2023 Felix Zailskas <felixzailskas@gmail.com>
```
Comment (Tims777) [inline Pipfile:2]: Is there a # missing here?

File: README.md [hunk 4]
```diff
@@ -1,18 +1,115 @@
...
+
+[REUSE documentation](https://reuse.software/faq/)
```
Comment (Tims777) [inline README.md:92]: Perfect!

File: Pipfile (hunk 3 above)
Comment (felix-zailskas) [reply Pipfile]: Fixed

**Review by Tims777 — APPROVED** [review]

**Pull request merged**